qiita_pet/handlers/api_proxy/sample_template.py

```diff
@@ -23,6 +23,33 @@
 from qiita_pet.handlers.api_proxy.util import check_access, check_fp
 
 
+def sample_template_get_req(samp_id, user_id):
```

Comment:
Discussed offline with @squirrelo - this is still not fixed and will be fixed soon